src/database/dsls/object_dsl.rs

```diff
@@ -444,7 +444,18 @@ impl Object {
             .await?;
         Ok(())
     }
-
+    pub async fn batch_claim(
+        user_id: &DieselUlid,
+        objects: &Vec<DieselUlid>,
+        client: &Client,
+    ) -> Result<()> {
+        let query = "UPDATE objects 
+            SET data_class = ('CONFIDENTIAL'), created_by = $1 
```

Comment:
But the claimed objects do not meet the requirements for our definition of the data class 'Confidential'?

Reply:
You're right, this should be set to `PRIVATE`.